src/main/java/com/microsoft/azure/vmagent/AzureVMCloudRetensionStrategy.java

```diff
@@ -157,6 +158,7 @@ public DescriptorImpl getDescriptor() {
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
 
+    @Symbol("azureVMCloudRetentionStrategy") //  TODO evaluate impact of renaming class to fix the default symbol name
```

Comment:
depending on how hard it is to migrate the data could rename the class so that this line isn't needed to fix the typo in the class name

Reply:
I think this can be done separately / as a follow up

Reply:
Agree.